src/ralph/accounts/models.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from dj.choices import Country, Gender
```

Comment:
dj.choices is 3rd party package in the same way as django is, so imo it should be at the same level in imports